src/myspace/AdditionalInfo.jsx

```diff
@@ -1,21 +1,149 @@
-import React from "react";
+import React, { useState, useEffect, useContext } from "react";
+import { BASE_API } from "../config";
+import { AuthContext } from "../AuthContext";
+import "./MySpace.css";
+
 
 export default function AdditionalInfo() {
-    return (
-        <div className="container-fluid" id="profile">
-          <div className="top">
-            <h1>
-              This is the Private page for Member's Additional Info page
-            </h1>
-          </div>
-          <div className="middle">
-            <h2>The site is currently under construction...</h2>
+  const [errorMessage, setErrorMessage] = useState(null);
+  const [additionalInfo, setAdditionalInfo] = useState({});
+  const { access_token, user } = useContext(AuthContext);
+
+  const requestAdditionalInfo = {
+    method: "GET",
+    headers: {
+      "Authorization": `Bearer ${access_token}`,
+      "Accept": "application/json",
+      "Content-Type": "application/json",
+    },
+  };
+
+
+  let data = "";
+  useEffect(() => {
+    fetch(`${BASE_API}/user/additional_info`, requestAdditionalInfo)
+      .then(async response => {
+        data = await response.json();
+        if (response.ok) {
+          setAdditionalInfo(data);
+        } else {
+          throw new Error(data);
```

Comment:
we can look at this today in our call - https://www.geeksforgeeks.org/reject-vs-throw-promises-in-javascript/